PhoneNumberKit/ParseManager.swift

```diff
@@ -34,12 +34,17 @@ final class ParseManager {
         let region = region.uppercased()
         // Extract number (2)
 
-        var nationalNumber = numberString
+        // Convert number to latin
+        var latinNumberString = numberString
+        if #available(iOSApplicationExtension 9.0, *) {
```

Comment:
why `iOSApplicationExtension` instead of just `iOS`? What about macOS and the other supported platforms?

Reply:
I wasn't sure too... this is what Xcode suggested.  Should I change to just iOS ?

Reply:
I think so... looks like that's what's used elsewhere in the project.
https://docs.swift.org/swift-book/ReferenceManual/Attributes.html